Makefile

```diff
@@ -4,7 +4,7 @@ BUILD_FLAGS = -scheme $(SCHEME) -destination $(DESTINATION)
 SCHEME ?= $(TARGET)-$(PLATFORM)
 TARGET ?= Kickstarter-Framework
 PLATFORM ?= iOS
-RELEASE ?= beta
+RELEASE ?= itunes
```

Comment:
In our `deploy:` task we're doing checks against `beta` wanna keep those?

Reply:
Will remove 👍